apps/ledger-live-desktop/src/renderer/modals/Stake/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useCallback, useMemo } from "react";
+import { useDispatch } from "react-redux";
+import { useTranslation } from "react-i18next";
+import { Icon, Text } from "@ledgerhq/react-ui";
+import { useRampCatalog } from "@ledgerhq/live-common/platform/providers/RampCatalogProvider/index";
+import { getAllSupportedCryptoCurrencyTickers } from "@ledgerhq/live-common/platform/providers/RampCatalogProvider/helpers";
+import { Account, AccountLike } from "~/../../../libs/ledgerjs/packages/types-live/lib";
+import { closeModal, openModal } from "~/renderer/actions/modals";
+import { useProviders } from "~/renderer/screens/exchange/Swap2/Form";
+import Modal, { ModalBody } from "~/renderer/components/Modal";
+import Box from "~/renderer/components/Box";
+import EntryButton from "~/renderer/components/EntryButton/EntryButton";
+import CoinsIcon from "./assets/CoinsIcon";
+import { useHistory } from "react-router-dom";
+
+interface StakeModalProps {
+  account: AccountLike;
+  parentAccount?: Account;
+}
+
+const StakeModal = ({ account, parentAccount }: StakeModalProps) => {
```

Comment:
What do you think about change the name?
modals/NotAvailableStake/...
NotAvailableStakeModal, NotAvailableStakeModalProps,...

Reply:
Can we use NoFundsStakeModal? for me it's more accurate.

Reply:
Done